packages/sdk/src/types/TextStyleTypes.ts

```diff
@@ -61,49 +44,42 @@ export type TextStyleUpdateType = {
 
 export interface AppearanceProperties {
     blendMode?: string;
+    dropShadowColor?: string;
+    fillColor?: string;
     opacity?: string;
+    strokeColor?: string;
+    useDropShadow?: boolean;
     useFill?: boolean;
     useStroke?: boolean;
-    useDropShadow?: boolean;
-    fillColor?: string;
-    strokeColor?: string;
-    dropShadowColor?: string;
 }
 
 export interface SelectedTextStyle {
-    paragraphStyleId?: string;
+    alignToBaseLine?: boolean;
+    baselineShiftValue?: string;
     characterStyleId?: string;
+    color?: ColorUsageUpdate;
```

Comment:
```suggestion
    color?: DisplayColor;
```
This a different color type.